lib/models/forms/new_post_form.dart

```diff
@@ -35,7 +35,8 @@ class NewPostForm {
   final int? inReplyToId;
 }
 
-BlueskyText stringToBluesky(String? value) => BlueskyText(value ?? '');
+BlueskyText stringToBluesky(String? value) =>
+    BlueskyText(value ?? '', enableMarkdown: false);
```

Comment:
Maybe you have seen links in markdown format in clients such as deck.blue, but I don't know if you need this setting in SkyBridge, so I have disabled it by setting `enableMarkdown` to false.